renderers/chords_renderer.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const parseVerse = require('../verse.js')['parseVerse'];
+
+class ChordsRenderer {
+  constructor(chordChartJson) {
+    this.chordChartJson = chordChartJson;
+  }
+
+  createHtmlChordChart() {
+    const chordChartHtml = document.createElement('div');
+    chordChartHtml.className = 'chart';
+
+    this.chordChartJson.forEach((verse) => {
+      const verseDiv = document.createElement('div');
+      verseDiv.className = 'verse';
+
+      this.createListOfWrappedVoices(verse)
+        .forEach((event) => {
+          verseDiv.appendChild(event);
+        });
+
+      chordChartHtml.appendChild(verseDiv);
+    });
+
+    return chordChartHtml;
+  }
+
+  createListOfWrappedVoices(verse) {
+    // Assumes voices are sorted by index and grouped by voice.
+    const voiceElements = new Map();
+
+    verse.forEach((event) => {
+      if (!voiceElements.has(event.voice)) {
+        voiceElements.set(event.voice, new Map([['index', 0], ['parentElement', document.createElement('div')]]));
+
+        voiceElements.get(event.voice).get('parentElement').className = event.voice;
+      }
+
+      const voice = voiceElements.get(event.voice);
+      this.appendVoiceContentDiv(voice.get('parentElement'), event.content, event.index - voice.get('index'));
```

Comment:
This is where you'll want to transpose the event.content in the case that event.voice starts with 'c'

Reply:
Yup, seems reasonable. I was going to create another task to implement the transpose portion though. Figured we can get rendering in without transpose first, help keep changes a bit smaller XD